src/xdocs/config_imports.xml

```diff
@@ -1114,6 +1136,27 @@ public class InputIllegalImport { }
   &lt;/subpackage&gt;
 &lt;/import-control&gt;
         </source>
+        <p>
+          In the example below, only classes that end with "Panel", "View", or "Dialog" in the
```

Comment:
> only classes that end with ....

I do not like term "Class/class" usage, as you you check Interfaces and enums.

Jdk specification differentiate them, see https://docs.oracle.com/javase/specs/jls/se7/html/index.html
`8. Classes`
`9. Interfaces`
We have even more in nuances, as we speak about top level type,  but in reality do not really check for top level type, we just parse imports in file not for class.

Please review https://docs.oracle.com/javase/specs/jls/se7/html/jls-7.html#jls-7.3 ("7.3. Compilation Units") .
This is what we actually do - we do import control of compilation unit. So I propose to rename "class" to "unit".
Xdoc should clearly state that we apply rules for whole "compilation unit".

Reply:
@romani `Compilation Units` refers to package and imports too. See https://docs.oracle.com/javase/specs/jls/se7/html/jls-7.html#jls-7.3 .
> A compilation unit consists of three parts, each of which is optional:
    A package declaration (§7.4)
    import declarations (§7.5)
    Top level type declarations (§7.6)

Classes, interfaces, etc... are called `type declarations`. see 
https://docs.oracle.com/javase/specs/jls/se7/html/jls-7.html#jls-7.6
https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.1 and https://docs.oracle.com/javase/specs/jls/se7/html/jls-9.html#jls-9.1 .
> A top level type declaration declares a top level class type (§8) or a top level interface type (§9). 

To be accurate with JLS, we would have to call it `typeName` or `typeDeclarationName` or some variant as we already have package defined as a node. We could go on with naming to include `top level` since that is the most accurate, so like `topLevelTypeName`.
I still think `class` should be used as I don't see people understanding what is meant by type.

Please tell me which name you want me to use.

Reply:
When I come about 'unit' I looked at our confirmation samples http://checkstyle.sourceforge.net/config_imports.html#ImportControl

So there are terms 'import-control' that just define package of validation and 'subpackage' that actually define packages to validate. Package is folder. So configuration is all about validation rules for files in folders, not classes only. Files are compilation units. What you are doing is actually defining import rules for certain file(compilation unit).
To my mind we have two options for names:
File
Unit

Both are fine, I just have more preference to unit, as we skipp non Java compiled files(properties or ...). So 'file' term is not very good for us here.

Reply:
Done.